.github/workflows/node.js.yaml

```diff
@@ -0,0 +1,24 @@
+name: Node.js CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x, 15.x]
```

Comment:
Any specific reason? We never said we're going to support all these versions? take one maybe latest and let's support that? If there is no requirement then why run the pipeline parallelly for all? @pcgamer1

Reply:
Took the latest version.